<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>75</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="tinjLabel">
          <property name="text">
           <string>tinj = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tinjButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Enter Injection Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>G-Function</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Sqrt Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveJpgButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save jpg</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCustomPlot" name="tprPlot" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>320</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="minRangeButtons">
        <item>
         <widget class="QPushButton" name="xminButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>x min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="y1minButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>y1 min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="y2minButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>y2 min</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="maxRangeButtons">
        <item>
         <widget class="QPushButton" name="xmaxButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>x max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="y1maxButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>y1 max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="y2maxButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>y2 max</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
